### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by fontexD** [dani-garcia/vaultwarden#3244] [Ideas]
could it be possible to implement a feature to store secret files like certificates in vaultwarden :=) ?

could be amazing!

**Comment by kpfleming** [comment]
It is already possible to store files using the Bitwarden clients. They can be attached to other items (logins) or stored as secure notes.

**Reply by XDavidT** [reply]
Which client support that?

**Reply by BlackDex** [reply]
All clients

**Comment by RealOrangeOne** [comment]
It's also worth noting that features like this would need to be supported by Bitwarden first, before they could be supported by Vaultwarden. Feature requests like that would be better made upstream.
